packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -3194,6 +3209,7 @@ void main() {
 //    properties.add(IntProperty('maxLines', maxLines, defaultValue: null));
 //    properties.add(EnumProperty<TextAlign>('textAlign', textAlign, defaultValue: null));
 //    properties.add(EnumProperty<TextDirection>('textDirection', textDirection, defaultValue: null));
+//    properties.add(DoubleProperty('textScaleFactor', textScaleFactor, defaultValue: null));
```

Comment:
I think i accidentally include this when I wrote the test, can you remove all the comments?

Reply:
Done